superset-frontend/plugins/plugin-chart-echarts/src/Graph/transformProps.ts

```diff
@@ -258,7 +282,9 @@ export default function transformProps(
       source: sourceNode.id,
       target: targetNode.id,
       value,
-      lineStyle: {},
+      lineStyle: {
+        color: firstColor,
```

Comment:
Could we use `sourceNodeColor` here? I feel that helps distinguish the source/target better than just using `firstColor`